<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>CURRICULUM VITAE</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">My Website</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="cv.html">Resume</a>
</li>
<li>
  <a href="repo.html">Repository</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">CURRICULUM VITAE</h1>

</div>


<div id="personal-information" class="section level2">
<h2>Personal Information</h2>
<p>Name: Jing Li</p>
<p>Address: 1229 Florida Ave. Unit306, Ames, IA, 50014</p>
<p>Phone: 334-728-5280</p>
<p>E-mail: <a href="jingli@iastate.edu">linked phrase</a></p>
</div>
<div id="academic-background" class="section level2">
<h2>Academic Background</h2>
<div id="education" class="section level3">
<h3>Education</h3>
<p>Sep 2011 - Jun 2015 Bachelor of Agriculture</p>
<p>College of Agriculture and Biotechnology China Agricultural University, Beijing, China</p>
<p>Aug 2014- May 2016 Master of Science</p>
<p>Department of Crop, Soil, and Environmental Sciences Auburn University, Auburn, AL, USA</p>
<p>Aug 2016 – Now PhD student</p>
<p>Interdepartmental Genetics Program Iowa State University</p>
</div>
<div id="research-experience" class="section level3">
<h3>Research Experience</h3>
<p> May 2013- May 2014. Undergraduate Research Practice. China Agricultural University.</p>
<p>Explore the effects of substrate moisture on the storage quality of cucumber seedlings in hardening stage.</p>
<p> Sep 2013- Apr 2014. Undergraduate Research Practice. China Agricultural University.</p>
<p>Validate the function of resistance-like genes of bacterial spot in tomato.</p>
<p> May 2015- May 2016. Graduate Research and Thesis. Auburn University.</p>
<p>Population structure and marker-trait association analysis of the U.S. peanut mini-core germplasm collection for resistance to tomato spotted wilted virus.</p>
<p> Aug 2016 – Oct 2016. Rotation Research. Iowa State University.</p>
<p>Identify orphan genes in legume family.</p>
<p> Oct 2016 – Dec 2016. Rotation Research. Iowa State University.</p>
<p>RNA seq and GO enrichment analysis for HC toxin response in maize.</p>
<p> Jan 2017 – Mar 2017. Rotation Research. Iowa State University.</p>
<p>Growth rate analysis of maize.</p>
<p> Mar 2017 – Now. Iowa State University.</p>
<p>Identify orphan gene and explore their function using RNA-seq data in yeast and maize.</p>
</div>
<div id="professional-skills" class="section level3">
<h3>Professional Skills</h3>
<p> Plant specific-Pollinating, phenotyping, genotyping, breeding</p>
<p> Wet lab skills: PCR, extra DNA and RNA, gel electrophoresis, protein purification, immunoprecipitation, cloning, virus and bacteria inoculation, physiological measurements</p>
<p> Bioinformatics skills: construct linkage mapping and association mapping, sequence alignment and annotation, RNA seq analysis, phylogenetic analysis</p>
<p> Microsoft Office, bash language, python, perl</p>
<p> Proficient in statistical software: R and SAS.</p>
</div>
<div id="teaching-experience" class="section level3">
<h3>Teaching Experience</h3>
<p> 2015 Spring TA Basic Soil Science (Undergraduate course) in Auburn University.</p>
<p> 2015 Fall TA Basic Soil Science (Undergraduate course) in Auburn University.</p>
<p> 2018 Spring TA Principles of Biology II (Undergraduate course) in Iowa State University.</p>
</div>
</div>
<div id="honor-awards" class="section level2">
<h2>Honor &amp; Awards</h2>
<p>2012 The Third Prize Scholarship</p>
<p>2012-2013 The Third Prize Scholarship</p>
<p>2013 The Third Prize in social practice research activity</p>
<p>2014 The First Prize of volleyball game in China Agricultural University.</p>
<p>2016 Outstanding International Student in Auburn University</p>
</div>
<div id="publications" class="section level2">
<h2>Publications</h2>
<ol style="list-style-type: decimal">
<li><p>Li J, Tang Y and Chen CY. 2016. Association Mapping of SSR Markers to TSWV Resistance in Cultivated Peanut. Proceedings of the 48th APRES Annual Meeting; Clearwater Beach, FL.</p></li>
<li><p>Li, J., Tang, Y., Jacobson, A.L., Dang, P.M., Li, X., Wang, M., Hagan, A. and Chen, C.Y., 2018. Population structure and association mapping to detect QTL controlling tomato spotted wilt virus resistance in cultivated peanuts. The Crop Journal.</p></li>
<li><p>Arendsee Z, Li J, Singh U, Seetharam A, and Wurtele ES. 2018. New software to identify and explore the orphan genes of maize. Proceedings of the 60th Annual Maize Genetics Conference; Saint-Malo, France.</p></li>
</ol>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
